### PR TITLE
feat: add installation tip

### PR DESCRIPTION
**Pull request by oranges-eating**
Signed-off-by: killer <1533063601@qq.com>

### CODE REVIEW

File: docs/Docker.md [hunk 1]
```diff
@@ -58,6 +58,30 @@ vim src/Conf.js
 ```
 Press **i**, modify serverUrl to http://your-ip:8000 (Casdoor front-end access address), modify clientId to the clientId of the application just added, modify appname to the set application name, and modify the organization to the set organization name. Click **Esc**, enter: wq to save and exit.
 ```shell
+vim src/auth/Auth.js
```
Comment (hsluoyz) [inline docs/Docker.md:61]: Don't assume all users are using vim, just describe it in general.

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: docs/Docker.md (hunk 1 above)
Comment (oranges-eating) [reply docs/Docker.md]: It has been modified .

File: docs/Docker.md [hunk 2]
```diff
@@ -57,6 +57,27 @@ cd web
...
+```shell
+export function getSignupUrl() {
+  return `${trim(authConfig.serverUrl)}/signup/${authConfig.appName}`;
+  //return getSigninUrl().replace(
+  // "/login/oauth/authorize",
+  // "/signup/oauth/authorize"
+  //);
+}
+```
+Click **Esc**, enter: wq to save and exit.
```
Comment (hsluoyz) [inline docs/Docker.md:80]: ?

Comment (oranges-eating) [reply docs/Docker.md]: sorry,it has been deleted

File: docs/Docker.md [hunk 3]
```diff
@@ -57,6 +57,27 @@ cd web
 vim src/Conf.js
```
Comment (hsluoyz) [inline docs/Docker.md:57]: @oranges-eating  Remove all vim stuff in next PR.

**Pull request merged**